samples/apps/copilot-chat-app/webapi/Controllers/BotController.cs

```diff
@@ -0,0 +1,254 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Memory;
+using SemanticKernel.Service.Config;
+using SemanticKernel.Service.Model;
+using SemanticKernel.Service.Skills;
+using SemanticKernel.Service.Storage;
+
+namespace SemanticKernel.Service.Controllers;
+
+[ApiController]
+public class BotController : ControllerBase
```

Comment:
We call these "Conversations" in the UX. Could we stay consistent with that terminology throughout?

Reply:
I understand your points. I wonder how that aligns with the concept of "bots are docs". hm... "conversations are docs" doesn't communicate the disruptive idea, IMO...